GameData/RealismOverhaul/RealPlume_Configs/RaiderNick/rn_us_probes.cfg

```diff
@@ -321,3 +321,84 @@
         }
     }
 }
+
+@PART[rn_kh9_ds]:FOR[RealPlume]:NEEDS[SmokeScreen]
+{
+    PLUME
+    {
+        name = Hypergolic-OMS-Red            //pre-fabbed plume you want
+        transformName = thrustTransform //which transform to attach the plume
+        localRotation = 0,0,0           //Optional - Any rotation needed
+        localPosition = 0,0,0.7           //Any offset needed
+        fixedScale = 1.8                  //Size adjustment to resize to engine
+        energy = 1                      //Adjust length of plume
+        speed = 1                       //Adjust speed to fit resize, 
+                                        //generally close to 1:1 with scale.
+    }
+    @MODULE[ModuleEngines*]
+    {
+        @name = ModuleEnginesRF
+    }
+    @MODULE[ModuleEngineConfigs]
+    {
+        %type = ModuleEnginesRF
```

Comment:
These don't really belong in Plume patching.  If they're necessary, put them in the main part.